scripts/class/Hero.js

```diff
@@ -5,6 +5,20 @@ class Hero {
     */
     constructor(level){
```

Comment:
Scrap `level`. We should pass the constructor a Hero-like object like {name: 'gohso', level: 4, ....} in order to construct a Hero object from json.
Basically we gonna have
```
constructor(hero) {
    hero = hero || {};

    this.name = hero.name || 'something random';
    this.level = hero.level || 1;
   ...
}
```